util/activation_memory_profiler.py

```diff
@@ -0,0 +1,134 @@
+import json
+from dataclasses import dataclass, field
+from typing import List
+
+import executorch.exir.memory as memory
+import torch
+from executorch.exir import ExecutorchProgramManager
+from executorch.exir.memory_planning import get_node_tensor_specs
+from executorch.exir.print_program import _stacktrace_to_framelist
+from executorch.exir.tensor import num_bytes_from_shape_and_dtype, TensorSpec
+from torch._export import ExportedProgram
+
+
+@dataclass
+class Allocation:
+    name: str
+    op_name: str
+    memory_id: int
+    size_bytes: int
+    fqn: str
```

Comment:
what's fqn stands for?

Reply:
Fully qualified name: like my_top_module.foo.linear_layers[0].linear